src/core/namespace.c

```diff
@@ -927,6 +966,42 @@ static int append_protect_system(MountList *ml, ProtectSystem protect_system, bo
         }
 }
 
+static int append_private_bpf(
+                MountList *ml,
+                PrivateBPF private_bpf,
+                bool protect_kernel_tunables,
+                bool ignore_protect,
+                const NamespaceParameters *p) {
+
+        int r;
+
+        assert(ml);
+
+        switch (private_bpf) {
+        case PRIVATE_BPF_NO:
+                if (protect_kernel_tunables)
+                        return append_static_mounts(ml, private_bpf_no_table, ELEMENTSOF(private_bpf_no_table), ignore_protect);
+                return 0;
+        case PRIVATE_BPF_YES:
+                r = bpffs_finalize(ml, p->bpffs_socket_fd);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to mount bpffs in bpffs_finalize(): %m");
+
+                r = pidref_wait_for_terminate_and_check("(sd-bpffs)", p->bpffs_pid, /* flags = */ 0);
+                TAKE_PIDREF(*p->bpffs_pid);
+                if (r < 0)
+                        return log_debug_errno(r, "Failed to pidref_wait_for_terminate_and_check: %m");
```

Comment:
the log message is slightly borked?
Maybe something `Failed to wait for subprocess configuring bpffs` ?

BTW, this is a debugging log, so not a big matter, but pidref_wait_for_terminate_and_check() logs all failures. Hence the logging here and below is not necessary.